proiect/src/app/ActionsService.java

```diff
@@ -0,0 +1,225 @@
+package app;
```

Comment:
maybe rename the app package to "service" and keep Main class outside. there is a general practice to use service, entity, repository packages and Main class for starting the app outside

Reply:
Resolved in issue #8